src/java/org/apache/cassandra/service/accord/fastpath/ParameterizedFastPathStrategy.java

```diff
@@ -64,7 +64,7 @@ static class WeightedDc implements Comparable<WeightedDc>
             public void serialize(WeightedDc dc, DataOutputPlus out, Version version) throws IOException
             {
                 out.writeUTF(dc.name);
-                out.writeUnsignedVInt(dc.weight);
+                out.writeUnsignedVInt32(dc.weight);
```

Comment:
by actually trying to serialize this I found this bug... why do we keep `out.writeUnsignedVInt` around if we don't allow it anymore?  🤷